TASK/TaskGoToCoordAnyMeansExtraParams.md

```diff
@@ -5,24 +5,24 @@ ns: TASK
 
 ```c
 // 0x1DD45F9ECFDB1BC9 0x094B75EF
-void TASK_GO_TO_COORD_ANY_MEANS_EXTRA_PARAMS(Ped ped, float x, float y, float z, float speed, Any p5, BOOL p6, int walkingStyle, float p8, Any p9, Any p10, Any p11);
+void TASK_GO_TO_COORD_ANY_MEANS_EXTRA_PARAMS(Ped ped, float x, float y, float z, float fMoveBlendRatio, Vehicle vehicle, BOOL bUseLongRangeVehiclePathing, int drivingFlags, float fMaxRangeToShootTargets, cs_type(Any) float fExtraVehToTargetDistToPreferVeh, cs_type(Any) float fDriveStraightLineDistance, int iExtraFlags, cs_type(Any) float fWarpTimerMS);
 ```
 
-```
-NativeDB Added Parameter 13: Any p12
-```
```

Comment:
You cannot add arguments to the function declaration, it will break the ABI. I will fix this for you 🙂